src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/blocks/HologramProjector.java

```diff
@@ -89,14 +109,116 @@ private static void openEditor(Player p, Block projector) {
+            hologram.teleport(l);
+
+            BlockStorage.addBlockInfo(projector, X_OFFSET_PARAMETER, String.valueOf(xOffset));
+            openEditor(pl, projector);
+            return false;
+        });
+
+        menu.addItem(13, new CustomItem(SkullItem.fromHash("2ae425c5ba9f3c2962b38178cbc23172a6c6215a11accb92774a4716e96cada"),
+            "&7X Offset (-): &e" + DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), X_OFFSET_PARAMETER))),
+            "", "&eLeft Click: &7-0.1", "&eRight Click: &7-0.5"));
+        menu.addMenuClickHandler(13, (pl, slot, item, action) -> {
+            double yOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER));
+            double xOffset = DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), X_OFFSET_PARAMETER)) + (action.isRightClicked() ? -0.5F : -0.1F));
+            double zOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Z_OFFSET_PARAMETER));
+            ArmorStand hologram = getArmorStand(projector, true);
+            Location l = new Location(projector.getWorld(), projector.getX() + xOffset, projector.getY() + yOffset, projector.getZ() + zOffset);
+            hologram.teleport(l);
+
+            BlockStorage.addBlockInfo(projector, X_OFFSET_PARAMETER, String.valueOf(xOffset));
+            openEditor(pl, projector);
+            return false;
+        });
+
+
+        // Z Axis
+        menu.addItem(6, new CustomItem(SkullItem.fromHash("2d9287616343d833e9e7317159caa2cb3e59745113962c1379052ce478884fa"),
+            "&7Z Offset (+): &e" + DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Z_OFFSET_PARAMETER))),
+            "", "&eLeft Click: &7+0.1", "&eRight Click: &7+0.5"));
+        menu.addMenuClickHandler(6, (pl, slot, item, action) -> {
+            double yOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Y_OFFSET_PARAMETER));
+            double xOffset = Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), X_OFFSET_PARAMETER));
+            double zOffset = DoubleHandler.fixDouble(Double.parseDouble(BlockStorage.getLocationInfo(projector.getLocation(), Z_OFFSET_PARAMETER)) + (action.isRightClicked() ? 0.5F : 0.1F));
+            ArmorStand hologram = getArmorStand(projector, true);
+            Location l = new Location(projector.getWorld(), projector.getX() + xOffset, projector.getY() + yOffset, projector.getZ() + zOffset);
+            hologram.teleport(l);
+
+            BlockStorage.addBlockInfo(projector, Z_OFFSET_PARAMETER, String.valueOf(zOffset));
+            openEditor(pl, projector);
+            return false;
+        });
+
+        menu.addItem(15, new CustomItem(SkullItem.fromHash("a3852bf616f31ed67c37de4b0baa2c5f8d8fca82e72dbcafcba66956a81c4"),
```

Comment:
You should NOT put texture hashes in here, place them into the HeadTexture enum to make them reusable.